### ISSUE TITLE
Change /coc/ to /community-guidelines/

### ISSUE THREAD

**Issue by jakearchibald** [GoogleChrome/devsummit#233]
Probably best to avoid any accusation of dick jokes here.

**Comment by jakearchibald** [comment]
Actually, `/community-guidelines/` makes more sense, as that's the link text.

**Comment by vschupmann** [comment]
BTW, Code of Conduct is the old name. The new name is Community Guidelines. Sometimes the websites will shorten this to "Guidelines" if there is a space issue.

**Comment by surma** [comment]
This landed with #237
